src/engine.cc

```diff
@@ -61,10 +61,12 @@ const OptionId kShowWDL{"show-wdl", "UCI_ShowWDL",
                         "Show win, draw and lose probability."};
 
 MoveList StringsToMovelist(const std::vector<std::string>& moves,
-                           bool is_black) {
+                           const ChessBoard& board) {
   MoveList result;
   result.reserve(moves.size());
-  for (const auto& move : moves) result.emplace_back(move, is_black);
+  for (const auto& move : moves) {
+    result.push_back(board.GetModernMove({move, board.flipped()}));
```

Comment:
Is there a reason for the change to `push_back`?

Reply:
Probably there's no difference in this case, but my thoughts were that there would be only one construction of Move (inside GetModernMove()) due to RVO, but with emplace_back I was not sure.

UPD. I've just checked on godbolt and indeed they generate identical code in this case, changing to emplace_back.